src/rules/html-closing-bracket-spacing.ts

```diff
@@ -0,0 +1,127 @@
+import { createRule } from "../utils"
+import type { AST } from "svelte-eslint-parser"
+
+export default createRule("html-closing-bracket-spacing", {
+  meta: {
+    docs: {
+      description: "Require or disallow a space before tag's closing brackets",
+      category: "Stylistic Issues",
+      conflictWithPrettier: true,
+      recommended: false,
+    },
+    schema: [
+      {
+        type: "object",
+        properties: {
+          startTag: {
+            enum: ["always", "never", "any"],
+          },
+          endTag: {
+            enum: ["always", "never", "any"],
+          },
+          selfClosingTag: {
+            enum: ["always", "never", "any"],
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
+    messages: {
+      expectedSpace: "Expected space before '>', but not found.",
+      unexpectedSpace: "Expected no space before '>', but found.",
+    },
+    fixable: "code",
+    type: "layout",
+  },
+  create(ctx) {
+    const options = {
+      startTag: "never",
+      endTag: "never",
+      selfClosingTag: "always",
+      ...ctx.options[0],
+    }
+    const src = ctx.getSourceCode()
+
+    /**
+     * Returns true if string contains newline characters
+     */
+    function containsNewline(string: string): boolean {
+      return /.*\n.*/s.test(string)
+    }
+
+    /**
+     * Returns true if string contains whitespace characters
+     */
+    function containsWhitespace(string: string): boolean {
+      return /.*\s.*/s.test(string)
+    }
+
+    /**
+     * Report
+     */
+    function report(
+      node: AST.SvelteStartTag | AST.SvelteEndTag,
+      shouldHave: boolean,
+    ) {
+      const tagSrc = src.getText(node)
+      const match = /(\s*)\/?>$/m.exec(tagSrc)
+
+      const end = node.range[1]
+      const start = node.range[1] - match![0].length
+      const loc = {
+        start: src.getLocFromIndex(start),
+        end: src.getLocFromIndex(end),
+      }
+
+      ctx.report({
+        loc,
+        messageId: shouldHave ? "expectedSpace" : "unexpectedSpace",
+        *fix(fixer) {
+          if (shouldHave) {
+            yield fixer.insertTextBeforeRange([start, end], " ")
+          } else {
+            const spaces = match![1]
+
+            yield fixer.removeRange([start, start + spaces.length])
+          }
+        },
+      })
+    }
+
+    return {
+      "SvelteStartTag, SvelteEndTag"(
+        node: AST.SvelteStartTag | AST.SvelteEndTag,
+      ) {
+        if (node.type === "SvelteEndTag" && options.endTag === "any") return
+
+        const tagSrc = src.getText(node)
+        const selfClosing =
+          tagSrc.slice(
+            Math.max(tagSrc.length - 2, 0),
+            Math.max(tagSrc.length - 1, 0),
+          ) === "/"
+
+        const tagType =
+          node.type === "SvelteEndTag"
+            ? "endTag"
+            : selfClosing
```

Comment:
I think you can use `node.selfClosing`.

Reply:
Hmmm.  I checked on the parser demo site and each seems to return the following results:

```html
<img /> <!-- "selfClosing": true -->
<img> <!-- "selfClosing": false -->
```

https://ota-meshi.github.io/svelte-eslint-parser/

Can you share the template you tried, as there may be other causes?

Reply:
I'm not sure rn, I'll try to find what caused my issue

Reply:
Replacing "/" check with `node.selfClosing` doesn't break any tests, and I couldn't find any edge case where `startTag.selfClosing` would be `false` even if it was a self closing tag, so I think it was me mistaking something while adding this rule.